portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,106 @@
+hr {
+    width: 95%;
```

Comment:
Prefer alphabetical ordering throughout this file :) https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
done